app/routes/public/sessions.js

```diff
@@ -84,6 +87,21 @@ export default class SessionsRoute extends Route {
       });
     }
 
+    if (params.sessionType) {
+      const sessions = params.sessionType.split(',');
```

Comment:
@iamareebjamal is there a better way to retrieve session types? This fails when sesison types themseleves have a `,`